src/index.js

```diff
@@ -7,6 +7,11 @@ import styles from './styles.css'
 export default function Typical({ steps, loop, className, wrapper = "p" }) {
   const typicalRef = useRef(null);
   const Component = wrapper;
+  const classNames = [styles.typicalWrapper];
+  
+  if (className) {
+    classNames.push(className);
```

Comment:
I think we need use css modules so rename `styles.css` to `styles.module.css` for be sure will not be overwritten.

What do you think ?

Reply:
I'm not familiar with css modules, if you can point me to any documentation that will be great, otherwise, feel free to make the changes, I'll be more than happy to approve them :+1:

Reply:
I found this small documentation if you want : https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/